components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/cmd/SetupCmd.java

```diff
@@ -688,4 +691,53 @@ private void informRestVersiontoUser(String restVersion) {
                 "You are using REST version - " + restVersion + " of API Manager. (If you want to change this, go to "
                         + "<MICROGW_HOME>/conf/toolkit-config.toml)");
     }
+
+    /**
+     * Extracts the platform and runtime and copy related jars and balos to extracted runtime and platform
```

Comment:
Possible checkstyle issue
```suggestion
     * Extracts the platform and runtime and copy related jars and balos to extracted runtime and platform.
```